web/GestureHandler.js

```diff
@@ -206,13 +203,14 @@ class GestureHandler {
     }
   }
 
-  setView(ref) {
+  setView(ref, propsRef) {
     if (ref == null) {
       this.destroy();
       this.view = null;
       return;
     }
 
+    this._propsRef = propsRef;
```

Comment:
I think this shouldn't be underscored here to be consistent with other properties.

Reply:
Excuse me, I don't understand, please explain what's your idea I will do it.

Reply:
I'm just saying that the rest of the properties don't have `_` prefix, so I think this should be named `propsRef` instead of `_propsRef`